app/models/slate_config.py

```diff
@@ -25,12 +45,14 @@ class SlateConfigModel:
     # store loaded slate configs (loaded at app startup)
     SLATE_CONFIGS_BY_ID = {}
 
-    def __init__(self, slate_id: str, display_name: str, description: str, curator_topic_label=None, experiments=None):
+    def __init__(self, slate_id: str, display_name: str, description: str,
+                 curator_topic_label: Optional[CuratorTopic] = None, experiments=None):
         self.id = slate_id
         self.displayName = display_name
         self.description = description
-        self.curator_topic_label = Optional[curator_topic_label]
         self.experiments = experiments or []
+        if curator_topic_label:
+            self.curator_topic_label = curator_topic_label
```

Comment:
I am not sure what the intent here is, but in this setup `self.curator_topic_label` won't exist as an attribute if `curator_topic_label` is `None`, which means any attempt to access it will raise an AttributeError. If that is desired you should leave a comment as to why; normal behavior is to just assign `self.curator_topic_label = curator_topic_label`, which will default to `None`.

Reply:
ty.  made the change